bytes/.env-dist

```diff
@@ -29,3 +29,6 @@ QUEUE_URI=
 # Optional environment variables
 BYTES_LOG_FILE=  # Optional file with Bytes logs.
 BYTES_DATA_DIR=  # Root for all the data. A change means that you no longer have access to old data unless you move it!
+BYTES_METRICS_MOUNTPOINTS=["/"]  # The mountpoints to keep track of the available and used amount of bytes in /metrics
+BYTES_METRICS_CACHE_TTL_SECONDS=30  # Time to wait to invalidate the cache for raw files per organization
```

Comment:
maybe specify that its the metrics cache in the comment, I know it seems redundant, but you wont believe how easy it is to overlook the name of this setting.
Same goes for the next setting.

Reply:
Ah these should have been removed when I decided to postpone the caching discussion